### PR TITLE
Form mailer

### PR DESCRIPTION
**Pull request by Caroline-CD**

### CODE REVIEW

**Review by NacymeG — DISMISSED** [review]
Ok for me 🥇

File: .env.sample [hunk 1]
```diff
@@ -5,3 +5,10 @@ DB_USER=root
 DB_PASSWORD=root
 DB_SCHEMA=expressdatabase
 PRIVATE_KEY= superSecretString
+
```
Comment (Hacene010) [inline .env.sample]: empty line !

File: routes/contact.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      <p>Téléphone: ${phoneNumber}</p>
+      <p>Raison Sociale: ${businessName}</p>
+      <p>Entreprise: ${companyName}</p>
+      <h3>Message:</h3>
+      <p>${message}</p>
+      `,
+    },
+    (err, info) => {
+      if (err) console.error(err);
+      else console.log(info);
```
Comment (Hacene010) [inline routes/contact.js]: Avoid to console log information

**Review by Hacene010 — CHANGES_REQUESTED** [review]

**Review by NacymeG — APPROVED** [review]
Ok for me !

**Review by Hacene010 — APPROVED** [review]

**Pull request merged**